src/code/utils/js-plumb-diagram-toolkit.coffee

```diff
@@ -234,11 +234,38 @@ module.exports = class DiagramToolkit
     if opts.isTransfer
       @kit.importDefaults
         Connector: ["Flowchart", {}]
-        Anchor:    ["Continuous", { faces:["left","right"] }]
+
+    linkSource = opts.linkModel?.sourceNode
+    linkTarget = opts.linkModel?.targetNode
+    linkRelation = opts.linkModel?.relation
+
+    # transfer-modifier links attach to fixed locations on the left or right of the flow node
+    flowNodeModifierAnchors = [[0, 0.4, -1, 0, 8, 0], [1, 0.4, 1, 0, -8, 0]]
+    # flow links attach to fixed locations on the left or right of the flow node
+    flowNodeFlowAnchors = [[0, 0.6, -1, 0, 16, 0], [1, 0.6, 1, 0, -16, 0]]
+    # other links attach to three fixed locations on the top or bottom of the flow node or
+    # two fixed locations on the left or right of the flow node chosen to not overlap the others
+    flowNodeLinkAnchors = [[0.3, 0, 0, -1, 0, 12], [0.5, 0, 0, -1, 0, 12], [0.7, 0, 0, -1, 0, 12],
+                           [0, 0.25, -1, 0, 8, 0], [0, 0.75, -1, 0, 8, 0],
+                           [0.3, 1, 0, 1, 0, -10], [0.5, 1, 0, 1, 0, -10], [0.7, 1, 0, 1, 0, -10],
+                           [1, 0.25, 1, 0, -8, 0], [1, 0.75, 1, 0, -8, 0]]
+    # links to non-flow nodes link to locations assigned by jsPlumb on the left, top, or right faces
+    standardAnchors = ["Continuous", { faces:["top","left","right"] }]
```

Comment:
It probably would not be much of a performance improvement but it would reduce the size of the function if you defined all these constants (line 243-253) in the constructor as properties of the object.